py-polars/polars/internals/lazy_functions.py

```diff
@@ -1387,11 +1387,13 @@ def arange(
     """
     low = pli.expr_to_lit_or_expr(low, str_to_lit=False)
     high = pli.expr_to_lit_or_expr(high, str_to_lit=False)
-
     if eager:
-        df = pli.DataFrame({"a": [1]})
-        return df.select(arange(low, high, step).alias("arange"))["arange"]
-
+        return (
```

Comment:
We could write 

```python
pl.arange(low, high, step, eager=True)
```

Reply:
I hang my head in shame for not spotting that, lol... will update accordingly.